bin/lib/cli.js

```diff
@@ -0,0 +1,15 @@
+var program = require('commander')
+var packageJson = require('../../package.json')
+var loadInit = require('./init')
+var loadStart = require('./start')
+
+module.exports = function cli (server) {
```

Comment:
Could the function name be more descriptive? Because it doesn't return anything. Maybe `startCli` or something like this?